packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -24,12 +24,23 @@ import { ColorContribution } from '@theia/core/lib/browser/color-application-con
 import { ColorRegistry, Color } from '@theia/core/lib/browser/color-registry';
 import { TabBarToolbarContribution, TabBarToolbarItem, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
 import { FrontendApplicationContribution, FrontendApplication } from '@theia/core/lib/browser/frontend-application';
-import { MenuModelRegistry, MessageService, Mutable } from '@theia/core/lib/common';
+import { MessageService, Mutable } from '@theia/core/lib/common';
 import { FileDialogService, OpenFileDialogProps } from '@theia/filesystem/lib/browser';
 import { LabelProvider } from '@theia/core/lib/browser';
 import { VscodeCommands } from '@theia/plugin-ext-vscode/lib/browser/plugin-vscode-commands-contribution';
-import { VSXExtensionsContextMenu, VSXExtension } from './vsx-extension';
+import { MenuContribution, MenuModelRegistry } from '@theia/core/lib/common/menu';
+import { VSXExtension, VSXExtensionsContextMenu } from './vsx-extension';
+import { QuickPickItem, QuickPickService } from '@theia/core/lib/common/quick-pick-service';
+import { VSXRegistryAPI } from '../common/vsx-registry-api';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Better just to add it to the `package.json`? Ask Paul - he's the expert on extraneous dependencies :-)